src/ZivoeRewardsVesting.sol

```diff
@@ -350,6 +353,10 @@ contract ZivoeRewardsVesting is ReentrancyGuard, Context, ZivoeVotes {
     /// @param _rewardsToken The asset that's being distributed.
     /// @param reward The amount of the _rewardsToken to deposit.
     function depositReward(address _rewardsToken, uint256 reward) external updateReward(address(0)) nonReentrant {
+        require(
```

Comment:
Please note that this doesn't fix https://github.com/sherlock-audit/2024-03-zivoe-judging/issues/129,
as the rounding can remain even when depositors are honest

To totally fix this, one should keep the remainder of the div at l364:
```
reward.div(rewardData[_rewardsToken].rewardsDuration);
```
In a separate variable and add it again during next depositReward to the total reward to distribute